.eslintrc.js

```diff
@@ -22,11 +23,47 @@ module.exports = {
   ],
 
   'rules': {
-    'indent': ['error', 2, { 'SwitchCase': 1 }],
-    'quotes': ['error', 'single', { 'allowTemplateLiterals': true }],
-    'semi': ['error', 'always'],
-    'comma-dangle': ['error', 'always-multiline'],
     'arrow-parens': ['error', 'always'],
     'no-console': 0,
+
+    // Stylistic rules (automatically fixed on commit).
+    'array-bracket-spacing': ['error', 'never'],
+    'brace-style': ['error', '1tbs', { 'allowSingleLine': true }],
```

Comment:
allowSingleLine ... my eyes ... my eyes ...

I know @wilsonpage is a big fan, but me as an old and grumpy person just can say that sometimes it can cause confusions.